examples/classification/plot_lda_qda.py

```diff
@@ -1,9 +1,12 @@
 """
 ====================================================================
-Linear and Quadratic Discriminant Analysis with confidence ellipsoid
+Linear and Quadratic Discriminant Analysis with covariance ellipsoid
 ====================================================================
 
-Plot the confidence ellipsoids of each class and decision boundary
+The ellipsoids display the double standard deviation for each class.
+With LDA, the standard deviation is the same for all the classes,
+while each class has its own standard deviation with QDA.
+Plot the covariance ellipsoids of each class and decision boundary.
```

Comment:
I suppose it's clear what we are intending to do with the example. Removing it seems good?